src/jdk.jdwp.agent/share/native/libjdwp/threadControl.c

```diff
@@ -412,14 +412,19 @@ insertThread(JNIEnv *env, ThreadList *list, jthread thread)
             if (error != JVMTI_ERROR_NONE) {
                 EXIT_ERROR(error, "getting vthread state");
             }
+            if ((vthread_state & JVMTI_THREAD_STATE_ALIVE) == 0) {
+              // Thread not alive so put on otherThreads list instead of runningVThreads.
+              // It might not have started yet or might have terminated. Either way,
+              // otherThreads is the place for it.
+              list = &otherThreads;
```

Comment:
Minor nit but the native code here uses 4-space indentation.

Reply:
fixed